.git-commit-template

```diff
@@ -0,0 +1,18 @@
+# Summarize the change
+# |<----   Preferably using up to 50 chars   --->|<------------------->|
+
+# (Optional) Explain why this change is being made
+# |<----   Try to limit each line to a maximum of 72 characters   ---->|
+
+# To include this change in changelog, uncomment and fill in the following:
```

Comment:
> > in **the** changelog

Fixed.

> Also, personally I feel it would be good to clarify we're talking about the released rpm package changelog?

Good point, added.

> And, should we also mention that msg and type are mandatory and resolves/related are optional or is it clear enough?

We haven't discussed this, but I think that all of the fields can be optional. For example, when you fix bug such as https://bugzilla.redhat.com/show_bug.cgi?id=1411575 , I don't think it needs to be mentioned in the changelog.

But maybe the type should be required when the message is present?

Reply:
> We haven't discussed this, but I think that all of the fields can be optional. For example, when you fix bug such as https://bugzilla.redhat.com/show_bug.cgi?id=1411575 , I don't think it needs to be mentioned in the changelog.
> 
> But maybe the type should be required when the message is present?

Right, true. I would guess the type and message need to go together for the changelog parser to work properly? It would be good to make it clear, not sure if you can put it in there and still keep it nice and clean...

Reply:
I am fine with putting it into README, but I am not so sure about the template, because there it has to be on another line to fit in. But take a look.

Reply:
Yeah, I would leave it out of the template as well and keep it just in the README.

Reply:
Ok, I like it better when not included in the template, so I removed it again.